2021-01-24 14:48:39,319 - INFO - allennlp.common.params - random_seed = 13370
2021-01-24 14:48:39,320 - INFO - allennlp.common.params - numpy_seed = 1337
2021-01-24 14:48:39,320 - INFO - allennlp.common.params - pytorch_seed = 133
2021-01-24 14:48:39,320 - INFO - allennlp.common.checks - Pytorch version: 1.7.1
2021-01-24 14:48:39,320 - INFO - allennlp.common.params - type = default
2021-01-24 14:48:39,321 - INFO - allennlp.common.params - dataset_reader.type = vimarc
2021-01-24 14:48:39,321 - INFO - allennlp.common.params - dataset_reader.lazy = False
2021-01-24 14:48:39,321 - INFO - allennlp.common.params - dataset_reader.cache_directory = None
2021-01-24 14:48:39,321 - INFO - allennlp.common.params - dataset_reader.max_instances = None
2021-01-24 14:48:39,321 - INFO - allennlp.common.params - dataset_reader.manual_distributed_sharding = False
2021-01-24 14:48:39,321 - INFO - allennlp.common.params - dataset_reader.manual_multi_process_sharding = False
2021-01-24 14:48:39,321 - INFO - allennlp.common.params - dataset_reader.tokenizer = None
2021-01-24 14:48:39,322 - INFO - allennlp.common.params - dataset_reader.token_indexers.token_characters.type = characters
2021-01-24 14:48:39,322 - INFO - allennlp.common.params - dataset_reader.token_indexers.token_characters.namespace = token_characters
2021-01-24 14:48:39,322 - INFO - allennlp.common.params - dataset_reader.token_indexers.token_characters.character_tokenizer = <allennlp.data.tokenizers.character_tokenizer.CharacterTokenizer object at 0x7f4e4d0154f0>
2021-01-24 14:48:39,322 - INFO - allennlp.common.params - dataset_reader.token_indexers.token_characters.start_tokens = None
2021-01-24 14:48:39,322 - INFO - allennlp.common.params - dataset_reader.token_indexers.token_characters.end_tokens = None
2021-01-24 14:48:39,322 - INFO - allennlp.common.params - dataset_reader.token_indexers.token_characters.min_padding_length = 5
2021-01-24 14:48:39,322 - INFO - allennlp.common.params - dataset_reader.token_indexers.token_characters.token_min_padding_length = 0
2021-01-24 14:48:39,322 - INFO - allennlp.common.params - dataset_reader.token_indexers.tokens.type = single_id
2021-01-24 14:48:39,322 - INFO - allennlp.common.params - dataset_reader.token_indexers.tokens.namespace = tokens
2021-01-24 14:48:39,322 - INFO - allennlp.common.params - dataset_reader.token_indexers.tokens.lowercase_tokens = True
2021-01-24 14:48:39,323 - INFO - allennlp.common.params - dataset_reader.token_indexers.tokens.start_tokens = None
2021-01-24 14:48:39,323 - INFO - allennlp.common.params - dataset_reader.token_indexers.tokens.end_tokens = None
2021-01-24 14:48:39,323 - INFO - allennlp.common.params - dataset_reader.token_indexers.tokens.feature_name = text
2021-01-24 14:48:39,323 - INFO - allennlp.common.params - dataset_reader.token_indexers.tokens.default_value = THIS IS A REALLY UNLIKELY VALUE THAT HAS TO BE A STRING
2021-01-24 14:48:39,323 - INFO - allennlp.common.params - dataset_reader.token_indexers.tokens.token_min_padding_length = 0
2021-01-24 14:48:39,323 - INFO - allennlp.common.params - dataset_reader.context_length_limit = 400
2021-01-24 14:48:39,323 - INFO - allennlp.common.params - dataset_reader.question_length_limit = 50
2021-01-24 14:48:39,323 - INFO - allennlp.common.params - dataset_reader.skip_when_all_empty = ['context_span', 'question_span', 'addition_subtraction', 'counting']
2021-01-24 14:48:39,323 - INFO - allennlp.common.params - dataset_reader.skip_impossible_questions = False
2021-01-24 14:48:39,323 - INFO - allennlp.common.params - dataset_reader.instance_format = drop
2021-01-24 14:48:39,323 - INFO - allennlp.common.params - dataset_reader.transformer_model_name = bert-base-multilingual-cased
2021-01-24 14:48:39,323 - INFO - allennlp.common.params - dataset_reader.relaxed_span_match_for_finding_labels = True
2021-01-24 14:48:39,949 - INFO - allennlp.common.params - train_data_path = fixtures/vimarc.json
2021-01-24 14:48:39,949 - CRITICAL - root - Uncaught exception
Traceback (most recent call last):
  File "/home/haophancs/anaconda3/lib/python3.8/site-packages/allennlp/common/params.py", line 238, in pop
    value = self.params.pop(key)
KeyError: 'data_loader'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "train.py", line 43, in <module>
    train_fixture(f"fixtures/{model}/")
  File "train.py", line 24, in train_fixture
    train_model_from_file(config_file, serialization_dir)
  File "/home/haophancs/anaconda3/lib/python3.8/site-packages/allennlp/commands/train.py", line 170, in train_model_from_file
    return train_model(
  File "/home/haophancs/anaconda3/lib/python3.8/site-packages/allennlp/commands/train.py", line 236, in train_model
    model = _train_worker(
  File "/home/haophancs/anaconda3/lib/python3.8/site-packages/allennlp/commands/train.py", line 453, in _train_worker
    train_loop = TrainModel.from_params(
  File "/home/haophancs/anaconda3/lib/python3.8/site-packages/allennlp/common/from_params.py", line 597, in from_params
    return retyped_subclass.from_params(
  File "/home/haophancs/anaconda3/lib/python3.8/site-packages/allennlp/common/from_params.py", line 629, in from_params
    kwargs = create_kwargs(constructor_to_inspect, cls, params, **extras)
  File "/home/haophancs/anaconda3/lib/python3.8/site-packages/allennlp/common/from_params.py", line 199, in create_kwargs
    constructed_arg = pop_and_construct_arg(
  File "/home/haophancs/anaconda3/lib/python3.8/site-packages/allennlp/common/from_params.py", line 303, in pop_and_construct_arg
    popped_params = params.pop(name, default) if default != _NO_DEFAULT else params.pop(name)
  File "/home/haophancs/anaconda3/lib/python3.8/site-packages/allennlp/common/params.py", line 243, in pop
    raise ConfigurationError(msg)
allennlp.common.checks.ConfigurationError: key "data_loader" is required
